configuration/components/hydra.libsonnet

```diff
@@ -0,0 +1,70 @@
+local deployment = (import './hydra/deployment.libsonnet');
```

Comment:
I don't know what are the best practices regarding jsonnet, but I wanted to keep following what we already do in the project. So the component main file lives there in the `components` folder. I will wait a bit to see what others say about this too. Honestly, I don't have a preference.

Reply:
I think either format is ok as long as we are consistent so I would personally just follow the existing pattern is this PR to avoid additional unrelated diff and we can discuss about a restructure after?